packages/shell-api/src/database.spec.ts

```diff
@@ -145,6 +145,12 @@ describe('Database', () => {
       });
     });
 
+    describe('getName', () => {
```

Comment:
no test for getMongo?

Reply:
There's no test for `db.getMongo`, and even the test for `getName` is kind of stupid since it's just returning an attribute of the class. I didn't want to write tests just for the sake of having a test, if it doesn't test anything worthwhile.

Reply:
I can add the test if you like

Reply:
Shouldn't hurt, is part of the spec, takes no time to add.

Reply:
Sure, although I disagree that it's useful :)